### ISSUE TITLE
cascade_ws_asset\Cascade dumps on createAsset

### ISSUE THREAD

**Issue by marknokes** [wingmingchan/php7-cascade-ws-ns#2]
Hi, Wing!!! I noticed in cascade_ws_ns7_8.8 that on creating an asset, the debug utility dump method was called. I had a look, and in my copy of the code I added the following on line 3591 to match your convention:

if( self::DEBUG ) { u\DebugUtility::dump( $std ); }

I hope you're well!

Thanks,

Mark

**Comment by wingmingchan** [comment]
Thanks. I added the code when I was working on REST. I will make sure
that the line won't output anything.

How are you? After the new design was launched, I hope that you are not
as busy.

Wing

>>> Mark Nokes <notifications@github.com> 03/02/18 10:19 AM >>>
Hi, Wing!!! I noticed in cascade_ws_ns7_8.8 that on creating an asset,
the debug utility dump method was called. I had a look, and in my copy
of the code I added the following on line 3591 to match your convention:
 if( self::DEBUG ) { u\DebugUtility::dump( $std ); }
 I hope you're well!
 Thanks,
 Mark
 —
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.
     
NaN.

**Comment by wingmingchan** [comment]
I checked the code.  That line has been commented out in the newest
version which works for both SOAP and REST.

Wing



>>> Mark Nokes <notifications@github.com> 03/02/18 10:19 AM >>>
Hi, Wing!!! I noticed in cascade_ws_ns7_8.8 that on creating an asset,
the debug utility dump method was called. I had a look, and in my copy
of the code I added the following on line 3591 to match your convention:
 if( self::DEBUG ) { u\DebugUtility::dump( $std ); }
 I hope you're well!
 Thanks,
 Mark
 —
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.
     
NaN.
